src/HtmlGenerator.Tests/GitBlameVisitorTests.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using GitGlyph;
+using LibGit2Sharp;
+using Microsoft.SourceBrowser.HtmlGenerator.Utilities;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace HtmlGenerator.Tests
+{
+    [TestClass]
+    public class GitBlameVistorTests
+    {
+        [TestMethod]
+        public void TestVisit()
+        {
+            string pathToTempDirectory = Path.GetTempPath() + "GitBlameVisitorRepo\\";
+            string pathToTestFile = pathToTempDirectory + "Test.cs";
+            string pathToGitBlameRepo = Directory.GetCurrentDirectory() + "\\Repositories\\GitBlameVisitorRepo.git";
+            Repository.Clone(pathToGitBlameRepo, pathToTempDirectory);
+            try

```

Comment:
the "try" should start just before Repository.Clone in order to clean the directory if the cloning fails.